packages/tupaia-web/src/features/Dashboard/Dashboard.tsx

```diff
@@ -200,13 +220,8 @@ export const Dashboard = () => {
           <StickyBar $isExpanded={isExpanded}>
             <TitleBar>
               <Title variant="h3">{title}</Title>
-              {activeDashboard && (
-                <ExportButton startIcon={<GetAppIcon />} onClick={() => setExportModalOpen(true)}>
-                  Export
-                </ExportButton>
-              )}
             </TitleBar>
-            <DashboardMenu activeDashboard={activeDashboard} dashboards={dashboards} />
+            <DashboardMenu activeDashboard={activeDashboard} dashboards={dashboards} setExportModalOpen={setExportModalOpen} isSubscribed={isSubscribed} handleSubscribeClick={handleSubscribeClick}/>
```

Comment:
With the `isSubscribed` being moved to the `activeDashboard` itself, and no longer managing the toast message here, I think we don't need to pass in a `handleSubscribeClick` function.

Ideally, we can create a component called `SubscribeButton` which will handle its own state and side effects. In the `ActionMenu` we just determine whether or not to render the button based on the `mailingListEnabled` flag